registrations/tests/test_signup_patch.py

```diff
@@ -122,41 +122,23 @@ def test_patch_extra_info_of_signup_with_empty_data(api_client, registration, si
     assert signup.extra_info == ""
 
 
+@freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test_registration_user_who_created_signup_can_patch_presence_status(
```

Comment:
Is there some reason to remove `test_registration_user_who_created_signup_can_patch_presence_status`? Maybe some duplicate test in some other file?

Reply:
The duplicate test existed in this particular file (still visible in the `main` branch: https://github.com/City-of-Helsinki/linkedevents/blob/main/registrations/tests/test_signup_patch.py)